lua/lspconfig/ui/lspinfo.lua

```diff
@@ -103,14 +103,16 @@ local function make_config_info(config, bufnr)
   return lines
 end
 
+---@param client table
+---@param fname string
 local function make_client_info(client, fname)
   local client_info = {}
 
   client_info.cmd = cmd_type[type(client.config.cmd)](client.config)
   local workspace_folders = fn.has 'nvim-0.9' == 1 and client.workspace_folders or client.workspaceFolders
   local uv = vim.loop
   local is_windows = uv.os_uname().version:match 'Windows'
-  fname = uv.fs_realpath(fname)
+  fname = vim.fn.fnamemodify(vim.fn.resolve(fname), ":p")
```

Comment:
Might be preferable to just do:

```suggestion
  fname = uv.fs_realpath(fname) or fname
```